index.js

```diff
@@ -1,39 +1,44 @@
 (function(root) {
+  var docEl = document && document.documentElement;
   var scrollbarSize;
+  var scrollTop;
 
   function getScrollbarSize() {
     if (typeof scrollbarSize !== 'undefined') return scrollbarSize;
 
-    var doc = document.documentElement;
```

Comment:
You could *cache* it above but the current line 2 actually looks for it as soon as the module is parsed. Besides, I think it's just a change better left to another PR.